fix account change event name

goktug-gurbuzturk-wp opened the pull request.

impredmet commented:
Approved. The only thing that would be interesting to add would be, after each `@ts-expect-error`, a brief description of why it's there. For example:

```typescript
// @ts-expect-error: This error is expected because the type is intentionally mismatched for testing purposes.
someFunctionThatCausesAnIntentionalError();
```

This can help future maintainers understand why the directive was used and avoid confusion or accidental removal.

goktug-gurbuzturk-wp commented:
> Approved. The only thing that would be interesting to add would be, after each `@ts-expect-error`, a brief description of why it's there. For example:
> 
> ```ts
> // @ts-expect-error This error is expected because the type is intentionally mismatched for testing purposes.
> someFunctionThatCausesAnIntentionalError();
> ```
> 
> This can help future maintainers understand why the directive was used and avoid confusion or accidental removal.

Thanks, oh sorry I guess the @ts-ignore is replaced by @ts-expect-error automatically as we moved to es6 and mistakenly pushed :) but I already fixed them in the [this commit](https://github.com/btc-vision/opwallet/pull/30/commits/5f06e792468ab6d1066c99bdeaf9ed261f8caefb) of typing issues PR and we can discuss it there
